File: app/src/main/java/com/jmalinen/blackjack/model/PlayerAction.kt
package com.jmalinen.blackjack.model

enum class PlayerAction(val displayName: String) {
    HIT("Hit"),
    STAND("Stand"),
    DOUBLE_DOWN("Double"),
    SPLIT("Split"),
    SURRENDER("Surrender"),
    INSURANCE("Insurance"),
    DECLINE_INSURANCE("No Insurance"),
    EVEN_MONEY("Even Money"),
    DECLINE_EVEN_MONEY("No Even Money")
}

File: ios/Blackjack/Blackjack/Model/PlayerAction.swift
enum PlayerAction: Hashable {
    case hit
    case stand
    case doubleDown
    case split
    case surrender
    case insurance
    case declineInsurance
    case evenMoney
    case declineEvenMoney

    var displayName: String {
        switch self {
        case .hit: "Hit"
        case .stand: "Stand"
        case .doubleDown: "Double"
        case .split: "Split"
        case .surrender: "Surrender"
        case .insurance: "Insurance"
        case .declineInsurance: "No Insurance"
        case .evenMoney: "Even Money"
        case .declineEvenMoney: "No Even Money"
        }
    }
}
